packages/core/src/page.js

```diff
@@ -49,17 +49,39 @@ export class Page {
   }
 
   // Go to a URL and wait for navigation to occur
-  async goto(url, { waitUntil = 'load' } = {}) {
+  async goto(url, { waitUntil = 'load', cookies } = {}) {
     this.log.debug(`Navigate to: ${url}`, this.meta);
 
     let navigate = async () => {
+      const mergeCookies = (userPassedCookie, autoCapturedCookie) => {
```

Comment:
Instead of keeping arrow function inside goto itslef let's create separate function outside this method as it makes this code clumsy.